src/ert/_c_wrappers/job_queue/queue.py

```diff
@@ -372,47 +374,67 @@ async def _queue_changes(
     async def _publish_changes(
         ens_id: str,
         changes,
-        ws_uri: str,
-        ssl_context: ssl.SSLContext,
-        headers: Mapping[str, str],
+        websocket: WebSocketClientProtocol,
     ):
         events = deque(
             [
                 JobQueue._translate_change_to_cloudevent(ens_id, real_id, status)
                 for real_id, status in changes.items()
             ]
         )
+        while events:
+            await asyncio.wait_for(websocket.send(to_json(events[0])), 60)
+            events.popleft()
 
-        retries = 0
-        while True:
+    async def _execution_loop_queue_via_websockets(
+        self,
+        ws_uri: str,
+        ens_id: str,
+        pool_sema: threading.BoundedSemaphore,
+        evaluators: List[Callable[..., Any]],
+        ssl_context,
+        headers,
+    ) -> None:
+        reconnects = 0
+        max_reconnects = 10
+        async for websocket in connect(
```

Comment:
The async itself contains reconnecting pattern, where to be honest to sure how to combine with the ert's one, ie. `reconnects` and `max_reconnects`.

Reply:
could you link to the documentation for the contained reconnect pattern? i could not find it : (

Reply:
It's in the code only https://github.com/python-websockets/websockets/blob/1bf9d1d766c80da4887240737266926f173fbcef/src/websockets/legacy/client.py#L594-L595

Reply:
sadly there seems to be no way offered by the API to hook into that code, nor any other way apparent to me.